packages/realm-server/tests/atomic-endpoints-test.ts

```diff
@@ -632,92 +630,6 @@ module(basename(__filename), function () {
             'the card source is correct',
           );
         });
-        test('writes will get file directly from disk enabling writes that rely on file system but not index', async function (assert) {
```

Comment:
this test now becomes too low level to test in this manner. the issue is that now we need to have meta for a card-def in order to even serialize an instance that uses it to disk. it is not possible to write an instance to disk if you dont have meta in the index for the necessary card def. which if why we are now batching out the `Realm.writeMany()` to assert that you always have index meta available to write instances to disk.

Reply:
moreover we no longer load code in the realm server--so i'm not sure if we even need the realm server form of this (worker only).